ui/k-toolbar/main.kytos

```diff
@@ -285,7 +285,10 @@ module.exports = {
     },
     get_queue_ids(){
       let values = Array(8).fill().map((_, i) => i)
-      var queue_ids = [{value: "", description: "none", selected: true}]
+      var queue_ids = [
+        {value: -1, description: "default", selected: false},
```

Comment:
Can we have the default selected here? Just so in the front-end by default it also follows the default? We've been using this way for [`spf_attribute`](https://github.com/kytos-ng/mef_eline/blob/master/ui/k-toolbar/main.kytos#L301)

Reply:
Changed.